### ISSUE TITLE
[en] wrong suggestion BEEN_PART_AGREEMENT[1]

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#3016]
> I was believe that it is correct.

I would have also expected "believing" to be offered

**Comment by MikeUnwalla** [comment]
'Believe' is a stative verb (https://learnenglish.britishcouncil.org/grammar/intermediate-to-upper-intermediate/stative-verbs). It is not usually used with in the present continuous.

An alternative correction is, "I believe that it is correct."

**Comment by tiff** [comment]
@MikeUnwalla thanks, then maybe this example is better?
> I was go there to find out ...

![Bildschirmfoto 2020-06-03 um 11 03 47](https://user-images.githubusercontent.com/37363/83617811-eb007600-a589-11ea-8fd3-7bd22e523e52.png)

**Comment by tiff** [comment]
Also I just realized that the POS-tagging prevents us from finding mistakes in sentences like:
> I was hope you could help me

"hope" is only "NN:UN". Can we solve this?

**Comment by MikeUnwalla** [comment]
> "hope" is only "NN:UN". Can we solve this?

Yes, but I think that each error must be fixed individually. Look at the 'lead' and 'hope' examples in disambiguation `was_is_VB_NN`.

I will try to fix the disambiguation for 'hope' later today (probably change the AP for #2544).

**Comment by tiff** [comment]
Same issue:
> If you're have trouble then ...

![Bildschirmfoto 2020-06-17 um 08 58 28](https://user-images.githubusercontent.com/37363/84865602-cbcd1280-b078-11ea-8e4d-3a67a6d4d79f.png)
